google/cloud/exceptions.py

```diff
@@ -21,6 +21,7 @@
 import json
 import six
 
+
```

Comment:
I'm fine with this extra line here, but it no longer has anything to do with your PR.